tutorials/inputs/controllers_gamepads_joysticks.rst

```diff
@@ -48,18 +48,24 @@ In Godot 4.0, there are 3 ways to get input in an analog-aware way:
  .. code-tab:: gdscript GDScript
 
     # `velocity` will be a Vector2 between `Vector2(-1.0, -1.0)` and `Vector2(1.0, 1.0)`.
+    # This handles deadzone in a correct way for most use cases.
+    # The resulting deadzone will have a circular shape as it generally should.
     var velocity = Input.get_vector("move_left", "move_right", "move_forward", "move_back")
 
-    # The line above is a shorter form of:
+    # The line below handles deadzone in a less optimal way. The resulting deadzone
+    # will have a square-ish shape when it should usually have a circular shape.
```

Comment:
This is how I would word it:

```suggestion
    # The line below is similar to `get_vector`, except that it handles the
    # deadzone in a less optimal way. The resulting deadzone will have a
    # square-ish shape when it should ideally have a circular shape.
```

\+ this should be changed in the C# comment as well.